pkg/registry/core/pod/strategy_test.go

```diff
@@ -267,53 +268,68 @@ func TestCheckGracefulDelete(t *testing.T) {
 	defaultGracePeriod := int64(30)
 	tcs := []struct {
```

Comment:
add `name string` to these test cases and document what condition each is testing so it's easy to follow why we expect the given output gracePeriod result